runatlantis.io/docs/repo-level-atlantis-yaml.md

```diff
@@ -264,11 +264,11 @@ version: 3
 abort_on_execution_order_fail: true
 projects:
 - dir: project1
-  execution_order_group: 2
-- dir: project2
   execution_order_group: 1
+- dir: project2
+  execution_order_group: 2
 ```
-With this config above, Atlantis runs planning/applying for project2 first, then for project1.
+With this config above, Atlantis runs planning/applying for project1 first, then for project2.
```

Comment:
I think this is not necessary, the previous comment basically enforces the fact that the order_group is taking precedence.

Reply:
@jamengual I get it. I will close this pull request.